torch/onnx/_internal/fx/passes/fx_to_onnxscript.py

```diff
@@ -305,17 +406,37 @@ def _export_fx_node_to_onnxscript(
 
             fx_name_to_onnxscipt_value[node.name] = output
             return
-
-        if node.target == operator.getitem:
-            # __getitem__ on Tensor or Sequence of tensors. Not tuple.
-            exporter_key = "getitem"
+        if (
+            isinstance(node.target, types.BuiltinFunctionType)
+            and node.target in function_dispatcher._BUILTIN_TO_EXPORTER_KEY_TABLE
+        ):
+            # This function expects built-in functions to have name like mul, mul_1, ...
+            # symbolic fx.graph contains built-in functions to calculate python values.
+            exporter_key = function_dispatcher._BUILTIN_TO_EXPORTER_KEY_TABLE[
+                node.target  # type: ignore[index]
+            ]
         elif (
             isinstance(node.target, torch._ops.OpOverload)
             and node.target in function_dispatcher._OP_OVERLOAD_TO_EXPORTER_KEY_TABLE
         ):
             exporter_key = function_dispatcher._OP_OVERLOAD_TO_EXPORTER_KEY_TABLE[
                 node.target
             ]
+        elif isinstance(node.target, torch._ops.OpOverloadPacket):
+            # aten::sym_size is the only OverloadPacket that we support.
+            # schema: aten::sym_size(Tensor self, int dim) -> Tensor
+            if node.target != torch.ops.aten.sym_size:
+                raise ValueError(
+                    f"Unsupported OverloadPacket: {node.target}, aten.sym_size is the only allowed OverloadPacket!"
```

Comment:
Great! As restrictive as possible in dev facing code (as convenient as possible in user facing code)!